views/js/pciCreator/dev/textReaderInteraction/pciCreator.js

```diff
@@ -23,6 +26,22 @@ define([
          * @returns {Object} Widget
          */
         getWidget : function () {
+            Widget.beforeStateInit(function (event, pci, state) {
+                if (pci.typeIdentifier && pci.typeIdentifier === "textReaderInteraction") {
+                    if (!pci.widgetRenderer) {
+                        pci.widgetRenderer = new Renderer({
+                            serial : pci.serial,
+                            $container : state.widget.$container,
+                            templates : {
+                                pages : pagesTpl,
+                                navigation : navigationTpl
+                            }
+                        });
+                    }
+                    pci.widgetRenderer.setState(state.name);
+                    pci.widgetRenderer.renderAll(pci.properties);
+                }
+            });
```

Comment:
Important note! This occurs only if you leave authoring section without reloading the page. However i guess it is obvious.